packages/x-components/src/x-modules/facets/entities/filter-entity.factory.ts

```diff
@@ -38,15 +38,39 @@ export class FilterEntityFactory {
    */
   protected modifiers: Record<Facet['id'], FilterEntityModifier[]> = {};
 
+  /**
+   * Contains the instantiated entities for each facet.
+   *
+   * @internal
+   */
+  protected cache: Record<Facet['id'] | '__unknown-facet__', FilterEntity> = {};
+
   /**
    * Creates a new FilterEntity from a filter.
    *
    * @param store - The {@link https://vuex.vuejs.org/guide/ | Vuex Store} that the entity will
    * use.
-   * @param filter - The filter necesary to know what Entity to create.
+   * @param filter - The {@link @empathyco/x-types#Filter | Filter} necessary to know what
+   * {@link FilterEntity} to create.
    * @returns The {@link FilterEntity} created by the factory.
    */
   createFilterEntity(store: Store<RootXStoreState>, filter: Filter): FilterEntity {
```

Comment:
this method doesn't create a FilterEntity anymore. Rename to `getFilterEntity`.

Therefore, rename the `instantiateFilterEntity` to `createFilterEntity`